src/drivers/src/uart_syslink.c

```diff
@@ -255,6 +256,10 @@ void uartslkResumeRx(void)
   NVIC_EnableIRQ(UARTSLK_IRQ);
 }
 
+void uartslkEnableIncoming() {
```

Comment:
Should we keep the same style in the file?

Reply:
Fixed